PKGBUILD

```diff
@@ -511,7 +513,13 @@ DEST_MODULE_LOCATION[3]="/kernel/drivers/video"' dkms.conf
     # 5.11
     if (( $(vercmp "$_kernel" "5.11") >= 0 )); then
       _kernel511="1"
-      _whitelist511=( 460* )
+      _whitelist511=( 460.27* 460.32* )
+      if [[ $pkgver != 460.32* ]]; then
+        cd "$srcdir"/"$_pkg"/kernel-$_kernel
+        msg2 "Applying 5.11-legacy.diff for $_kernel..."
```

Comment:
So this patch affects 460.27* and >460.32*. Is that correct?

Reply:
it than one is required for 460 .27, i fixed ti already with the nesting i did before
should i bother with drivers <460? (maybe good for vk dev)

Reply:
I'm not sure the current latest needs it but if it does, of course feel free to add it. Don't feel forced to do it, of course, that's not the point.

Reply:
latest vulkan dev driver now covered too